### PR TITLE
feat: migrate to opencerts-verify

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: src/constants/VerificationErrorMessages.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  [TYPES.IDENTITY]: {
+    failureTitle: "Certificate issuer identity is invalid",
+    successTitle: "Certificate issuer has been identified",
+    failureMessage: "This certificate was issued by an invalid issuer."
+  },
+  [TYPES.ADDRESS_INVALID]: {
+    failureTitle: "Certificate store address is invalid",
+    failureMessage:
+      "Please check that you have a valid smart contract with us and a correct certificate store address before proceeding."
```
[the remark this answers is not in the document]
Comment (Nebulis) [reply src/constants/VerificationErrorMessages.js]: done

[the remark this answers is not in the document]
Comment (rjchow) [reply src/constants/VerificationErrorMessages.js]: thanks boss

File: src/sagas/certificate.js [hunk 2]
```diff
@@ -45,378 +32,110 @@ const ANALYTICS_VERIFICATION_ERROR_CODE = {
...
-    yield put(
-      verifyingCertificateIssuerSuccess({
-        issuerIdentities: verificationStatuses
-      })
-    );
-    return true;
+    yield put({
+      type: types.VERIFYING_CERTIFICATE
+    });
+    const fragments = yield call(verify, certificate, { network: "ropsten" });
```
[the remark this answers is not in the document]
Comment (Nebulis) [reply src/sagas/certificate.js]: oooooops :D

thanks

File: src/config/index.js [hunk 3]
```diff
@@ -20,6 +20,8 @@ const GA_PRODUCTION_ID = "UA-130492260-1";
 const GA_DEVELOPMENT_ID = "UA-130492260-2";
 
 export const IS_MAINNET = publicRuntimeConfig.network === "mainnet";
+export const NETWORK_ID = IS_MAINNET ? "1" : "3";
+export const NETWORK_NAME = IS_MAINNET ? "homestead" : "ropsten";
```
[the remark this answers is not in the document]
Comment (Nebulis) [reply src/config/index.js]: done

File: src/sagas/certificate.js [hunk 4]
```diff
@@ -45,378 +32,110 @@ const ANALYTICS_VERIFICATION_ERROR_CODE = {
...
-export function* isValidSmartContract(storeAddress) {
-  const web3 = yield getSelectedWeb3();
-  const supportedContractHashes = [
-    "0x7135575eac76f1817c27b06c452bdc2b7e1b13240797415684e227def063a127"
-  ];
-  const onChainByteCode = yield web3.eth.getCode(storeAddress);
-  const hashOfOnChainByteCode = web3.utils.keccak256(onChainByteCode);
-  if (!supportedContractHashes.includes(hashOfOnChainByteCode)) {
-    throw new Error("Invalid smart contract: "`${storeAddress}`);
+    return {};
```
[the remark this answers is not in the document]
Comment (Nebulis) [reply src/sagas/certificate.js]: I added console, but I'm not sure what to do as well, we have nothing that is visible on the screen to dusplay those kind of errors :/

[the remark this answers is not in the document]
Comment (rjchow) [reply src/sagas/certificate.js]: logger is ok, user doesn't need to know about analytics errors

maybe one day we can stream logs back but i dont like the idea for now

File: src/reducers/certificate.js [hunk 5]
```diff
@@ -386,72 +198,15 @@ export function updateCertificate(payload) {
     payload
   };
 }
+export const verifyingCertificateSuccess = payload => ({
+  type: types.VERIFYING_CERTIFICATE_SUCCESS,
```
[the remark this answers is not in the document]
Comment (Nebulis) [reply src/reducers/certificate.js]: done

**Pull request merged**